app-libs/stf/src/helpers.rs

```diff
@@ -60,6 +60,7 @@ pub fn get_storage_double_map<K: Encode, Q: Encode, V: Decode + Clone>(
 	get_storage_by_key_hash(key)
 }
 
+//Get value in storage
 pub fn get_storage_by_key_hash<V: Decode>(key: Vec<u8>) -> Option<V> {
```

Comment:
Just a minor style issue on all three comments here:
1. A comment has a space between `///` and the comment -> `/// This is my comment.`
2. A comment starts with a capital letter
3. A comment has proper punctiation (e.g. ends with a period `.`)